chsdi/models/vector/bak.py

```diff
@@ -43,8 +45,17 @@ class UNESCO(Base, Vector):
     __bodId__ = 'ch.bak.schutzgebiete-unesco_weltkulturerbe'
     __label__ = 'bgdi_name'
     id = Column('bgdi_id', Integer, primary_key=True)
+    name_fr = Column('name_fr', Text)
     bgdi_name = Column('bgdi_name', Text)
-    bgdi_surface = Column('bgdi_surface', Text)
+    name_it = Column('name_it', Text)
+    name_en = Column('name_en', Text)
+    name_rm = Column('name_rm', Text)
```

Comment:
well you can use:
name_de = Column('bgdi_name', Text)`
we care only about the model properties. This will also allow you to make the changes i suggested in the mako.